app/dashboard/views.py

```diff
@@ -83,14 +83,24 @@ def new_interest(request, bounty_id):
     profile_id = request.session.get('profile_id')
     if not profile_id:
         return JsonResponse(
-            {'error': 'You must be authenticated!'},
+            {'error': 'You must be authenticated via github to use this feature!'},
             status=401)
 
     try:
         bounty = Bounty.objects.get(pk=bounty_id)
     except Bounty.DoesNotExist:
         raise Http404
 
+    num_issues = 3
+    active_bounties = Bounty.objects.filter(idx_status__in=['open', 'started'])
```

Comment:
Should we use `current().filter(...)` or `filter(current_bounty=True, ...)` ?

Reply:
this is why we code review what i wrote at 10pm with a glass of 🥃 in my hand :P

Reply:
ffcedbfaa204deb8c256e5a8a09f4d8cbe558981